sfge/src/main/java/com/salesforce/graph/ops/GraphUtil.java

```diff
@@ -181,11 +181,7 @@ private static List<Util.CompilationDescriptor> buildFolderComps(String sourceFo
                     "Could not read file/directory " + sourceFileVisitor.lastVisitedFile, ex);
         } catch (JorjeUtil.JorjeCompilationException ex) {
             throw new GraphLoadException(
-                    "Could not parse file "
-                            + sourceFileVisitor.lastVisitedFile
-                            + ": "
-                            + ex.getMessage(),
-                    ex);
+                    "Could not parse file " + sourceFileVisitor.lastVisitedFile, ex);
```

Comment:
Could you move this message to UserFacingMessages.java as well since the exception message would be displayed to the user?
We eventually need a large-scale change to clean up all the user-facing messages, but until then, we can at least move exception messages that are involved in individual work items.